posthog-node/test/posthog-node.spec.ts

```diff
@@ -152,6 +152,63 @@ describe('PostHog Node.js', () => {
     })
   })
 
+  describe('shutdown', () => {
```

Comment:
Although is this test meant to be in Node rather than core?

Reply:
yes 🙈

Reply:
oh, it's only core tests there, where shutdownAsync doesn't make too much sense 😓 . I'll keep it here for now! (vs. actually setting up coreStateless tests 🙈 )